README.md

```diff
@@ -93,6 +93,35 @@ Our React Native Android app now uses the `Hermes` JS engine which requires your
 
 ---
 
+# Testing Electron Auto-Update
+Testing the auto-update process can be a little involved. The most effective way to test this involves setting up your own release channel locally and making sure that you can notarize your builds.
+
+## Setting up Min.IO
+Rather than pushing new builds to the production S3 bucket, the best way to test locally is to use [Min.IO](https://min.io). Min.IO is an S3-compatible service that you can set up and deploy locally.
+
+Once you've set up a local Min.IO [install](https://docs.min.io/docs/minio-docker-quickstart-guide.html), the next step is to point your electron config at the Min.IO server. To do so, edit the `publish` block to look like:
+```
+publish: [{
+    provider: 's3',
+    endpoint: 'http://localhost:9000'
+    bucket: '<your-minio-bucket-name>',
+    channel: 'latest',
+}],
+```
+
+Note: while the `electron-updater` docs tell you to create a file named `dev-app-update.yaml`, this will **not** be helpful. Setting that file will, in development, tell the auto-updater where to look for builds. Unfortunately, on Mac the auto-updater will not install the new bits unless the app that is currently running is signed.
+
+Now, you need to upload a build. Before you can do so, you need to make sure that you can notarize builds. For this you will need an [Apple Developer](https://developer.apple.com) account. Go to the [Certificates, Identifiers, and Profiles](https://developer.apple.com/account/resources/certificates/list) page and create a new certificate for a Developer ID Application. Follow the instructions to create a Certificate Signing Request, and once the certificate has been created, add it to your keychain with the Keychain Access app.
+
+You will need to store your Apple ID in the `APPLE_ID` username and an [app-specific password](https://appleid.apple.com/account/manage) in the `APPLE_ID_PASSWORD` field. Entering your normal password will not work.
```

Comment:
It's not clear to me where this information should be stored/used. Do you mean in XCode somewhere? In the electron configuration? Could use some more instruction on this step

Reply:
Oh, I see ... it's used in `desktop/notarize.js`. Looks like those fields can be set as environment variables in the shell session where you run `npm run desktop-build` from, right?

Reply:
I ended up passing them like this:

```bash
APPLE_ID=<my_apple_id> \
APPLE_ID_PASSWORD=<my_app_specific_password> \
npm run desktop-build
```